src/app/components/ToggleField/ToggleField.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import Toggle from 'react-toggle';
+import 'react-toggle/style.css';
+import './ToggleField.css';
+
+export interface ToggleFieldProps {
```

Comment:
In this case I would probably just call this Toggle.  Reason being is that we've been using the term 'field' to represent components that interact with redux-forms